packages/newspringchurchapp/src/prayer/data/mutations/deleteSavedPrayer.js

```diff
@@ -0,0 +1,9 @@
+import gql from 'graphql-tag';
+
+export default gql`
+  mutation deleteSavedPrayer($nodeId: String!) {
```

Comment:
```suggestion
  mutation UnSavePrayer($nodeId: String!) {
```

Apollo best practices, I think we should backfill all our others too
https://www.apollographql.com/docs/react/essentials/mutations